docs/consume-packages/Package-Restore.md

```diff
@@ -1,242 +1,222 @@
+<!--[!INCLUDE [restore-dotnet-cli](includes/restore-dotnet-cli.md)]-->
 
 > [!IMPORTANT]
-> To add a missing package reference to the project file, use [dotnet add package](/dotnet/core/tools/dotnet-add-package?tabs=netcore2x), which also runs the `restore` command.
+> To add a missing package reference to the project file, use [dotnet add package](/dotnet/core/tools/dotnet-add-package), which also runs `restore`.
 
-## Restore using the nuget.exe CLI
+<a name="restore-using-the-nugetexe-cli"></a>
+## Restore by using the NuGet CLI
 
-[!INCLUDE [restore-nuget-exe-cli](includes/restore-nuget-exe-cli.md)]
+!!!USE INCLUDE TEXT from PR 2852!!!
 
-> [!IMPORTANT]
-> The `restore` command does not modify a project file or *packages.config*. To add a dependency, either add a package through the Package Manager UI or Console in Visual Studio, or modify *packages.config* and then run either `install` or `restore`.
-
-## Restore using MSBuild
-
-Use the [msbuild -t:restore](../reference/msbuild-targets.md#restore-target) command to restore packages listed in the project file (see [PackageReference](package-references-in-project-files.md)) and starting with MSBuild 16.5+, `packages.config` projects.
+<!--[!INCLUDE [restore-nuget-exe-cli](includes/restore-nuget-exe-cli.md)]-->
 
- This command is available only in NuGet 4.x+ and MSBuild 15.1+, which are included with Visual Studio 2017 and higher versions.
- Starting with MSBuild 16.5+, this command can also restore `packages.config` based projects when run with `-p:RestorePackagesConfig=true`.
+<a name="restore-using-msbuild"></a>
+## Restore by using MSBuild
 
-1. Open a Developer command prompt (In the **Search** box, type **Developer command prompt**).
+You can use [msbuild -t:restore](../reference/msbuild-targets.md#restore-target) to restore packages in NuGet 4.x+ and MSBuild 15.1+, which are included with Visual Studio 2017 and higher.
 
-   You typically want to start the Developer Command Prompt for Visual Studio from the **Start** menu, as it will be configured with all the necessary paths for MSBuild.
+This command restores packages in projects that use [PackageReference](package-references-in-project-files.md) for package references. Starting with MSBuild 16.5+, the command also supports [packages.config](/nuget/reference/packages-config) package references, when used with `-p:RestorePackagesConfig=true`.
 
-2. Switch to the folder containing the project file and type the following command.
+To use MSBuild restore:
 
-   ```cmd
-   # Uses the project file in the current folder by default
-   msbuild -t:restore
-   ```
+1. Open a Developer Command Prompt by searching for *developer command prompt* and starting the prompt from the Windows **Start** menu, which configures all the necessary paths for MSBuild.
 
-3. Type the following command to rebuild the project.
+1. Switch to the folder containing the project or *packages.config* file, and enter `msbuild -t:restore`.
```

Comment:
while msbuild.exe /t:restore does support packages.config restore through an opt in property `RestorePackagesConfig`, https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#restore-properties it only supports it in the context of a project, so just saying switching to the project folder is best.